src/components/ActiveWalletSection/index.tsx

```diff
@@ -153,16 +159,51 @@ export default function ActiveWalletSection() {
 
   const updateActiveWalletMutation = useMutation({
     mutationFn: async (walletId: number) => {
+      trackEvent({
+        event: 'Update Active Wallet Start',
+        properties: {
+          walletId,
+          ...sharedEventProperties
+        }
+      })
       await updateActiveWallet(walletId)
+      return walletId
+    },
+    onSuccess: (walletId) => {
+      invalidateQueries()
+      trackEvent({
+        event: 'Update Active Wallet Success',
+        properties: {
+          walletId,
+          ...sharedEventProperties
+        }
+      })
     },
-    onSuccess: invalidateQueries
+    onError: (error, walletId) => {
```

Comment:
do we want to log this error too? and send to sentry as well? like we do for `addGameplayWalletMutation` `onError`